index.html

```diff
@@ -3191,12 +3191,13 @@ <h2>Integrity of Related Resources</h2>
         </p>
 
         <p>
-Specification authors that write algorithms that fetch a resource based on the
-`id` of an object inside a [=conforming document=] need to consider whether
-that resource's content is vital to the validity of that document. If it is, the
-specification MUST produce a validation error unless the resource matches the
-expected media type and cryptographic digest.
+A [=conforming verifier implementation=] that makes use of a resource based on
+the `id` of an object inside a [=conforming document=] with a corresponding
+cryptographic digest MUST check the digest against the retrieved resource. If
```

Comment:
Just being picky...

```suggestion
the `id` of an object inside a [=conforming document=] with a corresponding
cryptographic digest appearing in a `relatedResource` object value MUST check the digest against the retrieved resource. If
```

It is pickiness. I will a accept either way.

Reply:
@iherman,

Hmm, this is in the related resource section. The `id` is in the `relatedResource` object, that's the object that is being referred to here, i.e., `relatedResource.id`, so I don't think your suggestion is correct and / or it is redundant in a strange way. IMO, if we need to, we can say "the `id` of a `relatedResource` object" instead of "the `id` of an object".